src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -435,6 +472,26 @@ public void setCreatedTime(Instant createdTime) {
         this.createdTime = createdTime;
     }
 
+    public Instant getDeletedTime() {
+        return deletedTime;
+    }
+
+    public void setDeletedTime() {
```

Comment:
Add a javadoc to this setter. It is not immediately clear that this sets the deleted time to now. If someone had to use this setter, they would need to click into the method to see the difference (which is annoying and not in the spirit of OOP)